server/bound_queries.cc

```diff
@@ -1620,6 +1666,34 @@ BoundGroupByQuery(const SelectExpression & select,
     //false means no implicit sort by rowhash, we want unsorted
     subSelect.reset(new BoundSelectQuery(subSelectExpr, from, alias, when, where, subOrderBy, calc, numBuckets));
 
+    std::vector<std::shared_ptr<ExpressionValueInfo> > groupInfo;
+    for (size_t c = 0; c < groupBy.clauses.size(); ++c) {
```

Comment:
Because the size of the calc vector is greater or equal to the size of the group by keys, since we also put aggregator's arguments in the calc vector, after the group by clauses.